readme.md

```diff
@@ -15,53 +15,26 @@ $ npm install unstated-debug
 
 ## Setup
 
-###### Before
+In the root of your app, import `unstated-debug`:
 
 ```js
-// AppContainer.js
-import {Container} from 'unstated';
-
-class AppContainer extends Container {
-	// …
-}
-```
-
-###### After
-
-```js
-// Container.js
-import {Container} from 'unstated';
-import unstatedDebug from 'unstated-debug';
-
-export default unstatedDebug()(Container);
-```
-
-```js
-// AppContainer.js
-import Container from './Container';
-
-class AppContainer extends Container {
-	// …
-}
+import React from 'react';
+import { render } from 'react-dom';
+import { Provide } from 'unstated';
+import App from './components/App';
+import UNSTATED from 'unstated-debug';
+
+UNSTATED.isEnabled = true;
+UNSTATED.logStateChanges = true;
```

Comment:
Why go this route rather than treating this like middleware?

Reply:
Cause middleware is stupid and people do stupid shit with it

Reply:
charming as ever 🎉